logsrvd/logsrvd.c

```diff
@@ -1264,7 +1308,7 @@ connection_closure_alloc(int sock)
  * Allocate a connection closure and send a server hello message.
  */
 static bool
-new_connection(int sock, struct sudo_event_base *base)
+new_connection(int sock, const union sockaddr_union *s_un, struct sudo_event_base *base)
```

Comment:
I would pass in const struct sockaddr *sa here.  The union is mostly just for storage in the caller.

Reply:
fixed